### PR TITLE
( The Cover-up-001 ) #ODevGate( Data dump) Board Member Ms Cassandra Willcox **Harassing** an OG's the 32end member to join ODevs.  

### PR DESCRIPTION
**Pull request by tmli3b3rm4n**
# #TheCoverUP #ODevGate

##  #FreeTimLieberman 

> Opposing views are real, they do exist and it's what science is made up of.

The rules are quite clear when it comes to **Harassing** other members of the community.  It clearly states that harassment is defined as  **Continued one-on-one communication after requests to cease continue.**  

**THE ALL TOO REAL SCENARIO:**
What does one do when the harasser and agitator are on the content review board?... They go public is what they do.   This PR is to make modifications to our Code of Conduct and create a new standard of rules for our board members.  
 
What's in this PR?...
**This does include the entire back and forth between Ms Willcox and me as promised.**  It also goes the extra mile in preventing these types of events from happening in the future.  

###  It's also calling for the removal and outing of *#ODevGate* harasser and *#CoverUpArtist* Ms Cassandra Willcox's.  Who thought by covering up and removing the unconvient truth of her actions it would go away.  

> Ms Willcox's wants to hold you to a higher standard then she holds herself and enforce that standard upon you.

### CODE REVIEW

**Comment by jorgehjr84** [comment]
Thank you for the PR, however at this time we find your claims to not have merit. We will be closing this PR now. Thank you again

**Comment by tmli3b3rm4n** [comment]
wow! that was weak sauce review.  The truth remains.

File: code-of-conduct.md [hunk 1]
```diff
@@ -31,9 +31,10 @@ Harassment includes, but is not limited to:
 - Sustained disruption of discussion
 - Unwelcome sexual attention, including gratuitous or off-topic sexual images or behaviour
 - Pattern of inappropriate social contact, such as requesting/assuming inappropriate levels of intimacy with others
-- Continued one-on-one communication after requests to cease
+- ** Continued one-on-one communication after requests to cease **
 - Deliberate “outing” of any aspect of a person’s identity without their consent except as necessary to protect others from intentional abuse
 - Publication of non-harassing private communication
+- No gender segragation ( this includes you ladies and gents ) No women or men only channels private or public.  Remember this is a community not a resort.
```
Comment (TheOneTheOnlyDavidBrown) [inline code-of-conduct.md]: s/segragation/segregation

Comment (tmli3b3rm4n) [reply code-of-conduct.md]: Actually I don't know why private channels are even allowed period.  That kind of breaks the no exclusive group rules.

Comment (ToxicBakery) [reply code-of-conduct.md]: >Harassment and other exclusionary behavior aren’t acceptable.

Private channels are not exclusionary behavior, they for information that does not fit well in public channels for a variety of valid reasons. For instance, why are you excluding me from your banking information? Why are you excluding me from the group of people that have keys to your home? These are ridiculous exclusion claims similar to arguing we should not have private channels. Maybe we should get rid of direct messages too. Why not just close down the slack as we are excluding everyone that can not afford a computer or an internet connection.

Comment (tmli3b3rm4n) [reply code-of-conduct.md]: lol, I don't think personal data or DM fall into the "exclusive" category.  Private channels create private exclusive groups of people.  They create places that take away from the community as a whole.

**Pull request closed without merging**